src/app/firebase.service.ts

```diff
@@ -1,15 +1,26 @@
 import { Injectable } from '@angular/core';
 import { AngularFirestore } from '@angular/fire/firestore';
-import { firestore } from 'firebase';
 import { Observable } from 'rxjs';
 
 @Injectable({
   providedIn: 'root'
 })
 export class FirebaseService {
 
-  constructor(public firestore: AngularFirestore) { }
+  currentSession: string;
+
+  constructor(public firestore: AngularFirestore) {
+    this.currentSession = 'kugTpWqJyrXaJZ4ZB6zE';
+  }
 
   allSessions$: Observable<any[]> = this.firestore.collection('sessions').valueChanges();
 
+  /**
+   * Adds student to firestore
+   * @param id Student' id
+   * @param studentInfo map of student's information including name
+   */
+  addStudent(id: string, studentInfo: { name?: string}) {
+    this.firestore.collection('sessions/' + this.currentSession + '/students').doc(id).set(studentInfo);
+  }
```

Comment:
Consider naming this `addStudentToSession` and taking a session ID as a parameter.